scrapenews/spiders/sabc.py

```diff
@@ -46,8 +47,10 @@ def parse_item(self, response):
             article_body = response.css('div.post-content')
             body_html = " ".join(article_body.css('::text').extract())
             byline = response.css('span.author::text').extract_first().strip()
-            publication_date_str = response.css('span.create::text').extract_first().strip()
-            publication_date = datetime.strptime(publication_date_str, '%d %B %Y, %I:%M %p')
+
+            # Ignore the first line which is a comment.
+            publication_date_str = response.xpath('//span[@class="create"]/text()').extract()[1]
+            publication_date = lib.parse_long_month_hour_min_meridian(publication_date_str)
```

Comment:
no, just no :)

Please. Let's revert everything back to hardcoded date format strings please. They're quick to ready and clear what they do. This is also irrelevant to this pull request.